ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -92,6 +92,7 @@ Item {
         CommunityIntroDialog {
             id: communityIntroDialog
 
+            destroyOnClose: false
```

Comment:
should we keep it alive?

Reply:
Yes, we can close & open it several times

Reply:
You can remove it now, the culprit has been eliminated :)

Reply:
kk, just rebased

Reply:
Forgot to push? ;)